modules/highgui/src/window_winrt.cpp

```diff
@@ -205,6 +205,18 @@ CV_IMPL int cvGetTrackbarPos(const char* trackbar_name, const char* window_name)
 
 /********************************** Not YET implemented API ****************************************************/
 
+CV_IMPL int cvPollKey()
+{
+    CV_WINRT_NO_GUI_ERROR("cvPollKey");
+
+    for (;;)
+    {
+        CvWindow* window;
+
+        // TODO: implement appropriate logic here
+    }
```

Comment:
It's obviously dead code.

Reply:
loop removed, comment remains as in adjacent functions.